wire-ios-sync-engine/Tests/Source/Use cases/OneOnOneConversationCreationStatusUseCaseTests.swift

```diff
@@ -0,0 +1,144 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+import WireTesting
```

Comment:
swiftlint warnings to resolve here

Reply:
do we really need this `sorted_imports` rule? seems overkill

Reply:
I didn't know we had it, I would expect that `swiftlint --fix`